packages/trimerge-sync/src/TrimergeClient.ts

```diff
@@ -135,6 +156,8 @@ export class TrimergeClient<
     this.mergeAllBranches = mergeAllBranches;
     this.computeRef = computeRef;
     this.addNewCommitMetadata = addNewCommitMetadata;
+    this.docCache =
+      docCache ?? new InMemoryDocCache<SavedDoc, CommitMetadata>();
```

Comment:
move to destructor

Reply:
Nice, was able to do that for migrate too.